app/components/blacklight_range_limit/range_form_component.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= form_tag search_action_path, method: :get, class: [@classes[:form], "range_#{@facet_field.key} d-flex justify-content-center"].join(' ') do %>
```

Comment:
Would you mind adding a comment above explaining why you've overridden this template?

Reply:
I don't mind, added.